lib/client.js

```diff
@@ -5,6 +5,7 @@
  * Dependencies
  */
 
```

Comment:
on the streams I used `this._ = PrivateStream()` since it's easy to type and it's common to have `_` as the start of private members..

 `this._.foo` also looks cool because `._.` is almost an emoji :D